db/migrate/20200712085216_create_users.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+class CreateUsers < ActiveRecord::Migration[6.0]
+  def change
+    create_table :users do |t|
+      t.string :first_name
+      t.string :last_name
+      t.string :email
+      t.string :password_digest
+      t.string :phone_number
+
```

Comment:
I think it would be better to store only the email and password in the user table and transfer all other data to the account table